x-pack/qa/smoke-test-plugins-ssl/build.gradle

```diff
@@ -54,6 +54,14 @@ project(':plugins').getChildProjects().each { pluginName, pluginProject ->
 testClusters.matching { it.name == "yamlRestTest" }.configureEach {
   testDistribution = 'DEFAULT'
   setting 'xpack.monitoring.collection.interval', '1s'
+  setting 'xpack.monitoring.collection.enabled', 'false' //will enable in test
+  setting 'xpack.monitoring.exporters._http.enabled', 'false'//will enable in test
+  setting 'xpack.monitoring.exporters._http.type', "http"
+  setting 'xpack.monitoring.exporters._http.host', "https://example.com" //will be replaced in test
+  setting 'xpack.monitoring.exporters._http.auth.username', 'monitoring_agent'
+  setting 'xpack.monitoring.exporters._http.ssl.verification_mode', 'full'
+  setting 'xpack.monitoring.exporters._http.ssl.certificate_authorities', 'testnode.crt'
+  keystore 'xpack.monitoring.exporters._http.auth.secure_password', 'x-pack-test-password'
```

Comment:
This test has been muted for 5+ years... I believe that this is needed due to the changes in secure settings and stricter validation. Should be functionally equivalent.